src/components/Auth.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function Auth() {
```

Comment:
Lets use arrow functions instead of function declaration

Instead of 
```JSX
export default function Auth() {}
```
use this
```JSX
export const Auth: FC<AuthProps> = (props) => {}
```